admin_guide/configuring_authentication.adoc

```diff
@@ -174,40 +192,78 @@ http://httpd.apache.org/docs/2.4/programs/htpasswd.html[`htpasswd`].
 
 === LDAP Authentication
 
-Set `*LDAPPasswordIdentityProvider*` in the `*identityProviders*` stanza to
-validate user names and passwords against an LDAP v3 server, using simple
-bind authentication.
-
-During authentication, the LDAP directory is searched for an entry that matches the provided username.
-If a single unique match is found, a simple bind is attempted using the distinguished name (DN) of the entry
-plus the provided password. Here are the steps taken:
+Set *LDAPPasswordIdentityProvider* in the `*identityProviders*` stanza to
+validate user names and passwords against an LDAPv3 server, using simple bind
+authentication.
+
+During authentication, the LDAP directory is searched for an entry that matches
+the provided user name. If a single unique match is found, a simple bind is
+attempted using the distinguished name (DN) of the entry plus the provided
+password. Here are the steps taken:
+
+. Generate a search filter by combining the attribute and filter in the
+configured `*url*` with the user-provided user name.
```

Comment:
Don't asterisks render visibly inside backticks? In this case, the `url` value is the name of the field... I didn't think we wanted asterisks to appear